_common

```diff
@@ -116,7 +116,8 @@ handle_unknown() {
         group_mailto=$(echo "$group_description" | perl -n -wE'm/.*MAILTO: (\S+).*/ and say $1' | head -1)
         group_mailto=${group_mailto:-$notification_address}
         if [[ -n "$group_mailto" ]]; then
-            echo "$header"$'\n'"Reason: $reason"$'\n'"$excerpt" | mailx -s "Unknown issue to be reviewed (Group $group_id)" -r "openqa-label-known-issues <$from_email>" "$group_mailto"
+            group_name=$(echo "$group_data" | runjq -r '.[0].name' ) || true
```

Comment:
I know it's not checked by shellcheck, but I would add the new variable to line 103 (`local ...`)

Reply:
And please see my earlier suggestion. I don't think we should ignore an error trying to parse the group name. We do that for the description which can be non-existant but we shouldn't for the name.